jsx.js

```diff
@@ -0,0 +1,11 @@
+var jsx = require('babel-plugin-transform-jsx');
+var h = require("./src/virtual-hyperscript");
+
+/**
+ * Constructor function for https://www.npmjs.com/package/babel-plugin-transform-jsx
+ * Docs are in ./src/cycle-dom
+ */
+module.exports = function (attrs) {
+   var convertedJSX = jsx(attrs);
```

Comment:
You don't need to require the module. You do the necessary transforms on the JSX object here. So for example:

```js
function (jsxObject) {
  return h(
    jsxObject.elementName,
    jsxObject.attributes,
    jsxObject.children
  )
}
```